cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -147,18 +147,30 @@ func getSkyMsg(ip string, port int) *skymsg.Service {
 	}
 }
 
-func (ks *kube2sky) generateRecordsForHeadlessService(subdomain string, e *kapi.Endpoints, svc *kapi.Service) error {
+func (ks *kube2sky) generateRecordsForHeadlessService(subdomain string, e *kapi.Endpoints, svc *kapi.Service, isNewStyleFormat bool) error {
 	for idx := range e.Subsets {
 		for subIdx := range e.Subsets[idx].Addresses {
-			subdomain := buildDNSNameString(subdomain, fmt.Sprintf("%d%d", idx, subIdx))
-			b, err := json.Marshal(getSkyMsg(e.Subsets[idx].Addresses[subIdx].IP, svc.Spec.Ports[0].Port))
+			recordLabel := fmt.Sprintf("%d%d", idx, subIdx)
```

Comment:
Add a comment that this value is meaningless and should just be treated like a random number